.github/workflows/qase.yml

```diff
@@ -0,0 +1,29 @@
+name: Qase Test Upload
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - '**/schemas/**'
+  
+jobs:
+  run-script:
+    name: update-qase
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
https://github.com/rancher/security-team/blob/main/docs/standards/rancher-gha-standards.md#security-recommendations

Reply:
you're right... I was basing off of the FOSSA action though directly from rancher/rancher! 🤯 https://github.com/rancher/rancher/blob/main/.github/workflows/fossa.yml#L18